modules/games/definitions/jesonmor/instructions.ts

```diff
@@ -0,0 +1,18 @@
+import { JesonmorDefinition } from "./_types";
+
+const jesonmorInstructions: JesonmorDefinition["instructions"] = {
+  startTurn: { line: ["Go!"] },
```

Comment:
This should be something like;

```suggestion
  startTurn: { line: ["Select", "which", "horses", "to act with"] },
```

(I want "Select"` as a single word since I have an idea about rendering that instruction differently somehow, but we don't do that yet)

(Having "horses" as a single word means it'll match the `horses` unit type, and be shown as such an icon in the current player's colour)

Reply:
Actually, it'd be nice to have a different instruction here when you're forced to oust an enemy from the center square! Something like:

```suggestion
  startTurn: {
    ifelse: [
      { overlaps: ["oppunits", "center"] },
      { line: ["Select", "a", "horses", "to oust", {unitat: { onlyin: "center" }}]  },
      { line: ["Select", "which", "horses", "to act with"] }
    ]
  },
```

Reply:
Ahh very cool 👍